src/1AF120.c

```diff
@@ -18,9 +18,9 @@ ApiStatus AddEffectOffset(Evt* script, s32 isInitialCall) {
     Bytecode* args = script->ptrReadPos;
     EffectInstance* effect = evt_get_variable(script, *args++);
 
-    effect->data->pos.x += evt_get_variable(script, *args++);
-    effect->data->pos.y += evt_get_variable(script, *args++);
-    effect->data->pos.z += evt_get_variable(script, *args++);
+    ((EffectInstanceData*)effect->data)->pos.x += evt_get_variable(script, *args++);
+    ((EffectInstanceData*)effect->data)->pos.y += evt_get_variable(script, *args++);
+    ((EffectInstanceData*)effect->data)->pos.z += evt_get_variable(script, *args++);
```

Comment:
Is it possible to put `effect->data` into a temp so it can be casted once instead

Reply:
No, it's grabbing it thrice and that make it only do so once